src/cloudfoundry_dashboards/cf_cells_capacity.json

```diff
@@ -656,6 +656,26 @@
         "tagsQuery": null,
         "type": "query",
         "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "${DS_PROMETHEUS}",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Bosh_job_name",
```

Comment:
Can you please rename this to just "Job"? We'll have consistent labels with other dashboards